### PR TITLE
More backports

### PR DESCRIPTION
**Pull request by portante**

### CODE REVIEW

File: agent/tool-scripts/postprocess/gold/proc-vmstat/csv/proc-vmstat_pgalloc_delta_sec.csv [hunk 1]
```diff
@@ -4,8 +4,8 @@ timestamp_ms,dma,dma32,movable,normal
 1424904539496.22,0,2379.14407641464,0,3121.66887987701
 1424904542498.77,0,979.833791995655,0,1874.73977401208
 1424904545501.49,0,1439.361659485,0,1657.16418732007
-1424904548503.95,0,742.724309070785,0,1710.59733246079
-1424904551507.05,0,1058.2397844415,0,1479.13817573604
+1424904548503.96,0,742.721832944019,0,1710.59162959663
+1424904551507.05,0,1058.24331171904,0,1479.14310593328
```
Comment (ndokos) [inline agent/tool-scripts/postprocess/gold/proc-vmstat/csv/proc-vmstat_pgalloc_delta_sec.csv:8]: If perl can't get two decimal places right, why maintain twelve decimal places? Not sure this is worth worrying about here, but in general, we need to worry about (false) accuracy.

File: agent/util-scripts/pbench-sysinfo-dump [hunk 2]
```diff
@@ -152,12 +157,12 @@ function collect_ara_data {
...
-	stockpile_local_remote_user=$(getconf.py local_remote_user stockpile)
-	stockpile_host_remote_user=$(getconf.py host_remote_user stockpile)
-	stockpile_output_path=$(getconf.py stockpile_output_path stockpile)
-	stockpile_log=$(getconf.py stockpile_log stockpile)
+	stockpile_path=$(pbench-config stockpile_path stockpile)
+	stockpile_user=$(pbench-config stockpile_user stockpile)
+	stockpile_local_remote_user=$(pbench-config local_remote_user stockpile)
+	stockpile_host_remote_user=$(pbench-config host_remote_user stockpile)
+	stockpile_output_path=$(pbench-config stockpile_output_path stockpile)
+	stockpile_log=$(pbench-config stockpile_log stockpile)
```
Comment (ndokos) [inline agent/util-scripts/pbench-sysinfo-dump:165]: Have we switched to `pbench-config` in 0.69? I thought we only did that for 0.70+.

**Review by ndokos — CHANGES_REQUESTED** [review]

File: agent/util-scripts/pbench-sysinfo-dump (hunk 2 above)
Comment (portante) [reply agent/util-scripts/pbench-sysinfo-dump:165]: Hmmm, I don't think we switched; I'll check and confirm here.

File: agent/tool-scripts/postprocess/gold/proc-vmstat/csv/proc-vmstat_pgalloc_delta_sec.csv (hunk 1 above)
Comment (portante) [reply agent/tool-scripts/postprocess/gold/proc-vmstat/csv/proc-vmstat_pgalloc_delta_sec.csv:8]: Definitely not worth worrying about here, as these metrics will be replaced by PCP and/or Prometheus in the future.

In general, the Perl change appears to have happened at Perl 5 v30.  But it is stable after v30 it seems.

File: agent/util-scripts/pbench-sysinfo-dump (hunk 2 above)
Comment (ndokos) [reply agent/util-scripts/pbench-sysinfo-dump:165]: Did you check this?

**Review by ndokos — CHANGES_REQUESTED** [review]

**Review by portante — COMMENTED** [review]
Thanks for reviewing, @ndokos, corrected the unwanted `pbench-config` change.

File: agent/util-scripts/pbench-sysinfo-dump (hunk 2 above)
Comment (portante) [reply agent/util-scripts/pbench-sysinfo-dump]: Fixed.

**Review by ndokos — APPROVED** [review]

**Review by riya-17 — APPROVED** [review]

**Review by dbutenhof — APPROVED** [review]

**Pull request merged**